sdk/keyvault/azure-keyvault-keys/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 4.2.0b1 (2020-03-10)
+- Support for Key Vault API version 7.1-preview
+([#10124](https://github.com/Azure/azure-sdk-for-python/pull/10124))
+  - Added `recoverable_days` to `CertificateProperties`
+  - Added `ApiVersion` enum identifying Key Vault versions supported by this package
```

Comment:
Hi! I think we also need to add `import` to the list of possible values for key operation.

Reply:
Thanks, I added it but didn't call it out in this changelog (have done now).